Services/Block/classes/class.ilColumnGUI.php

```diff
@@ -576,6 +578,11 @@ public function determineBlocks(): void
                     if ($side == IL_COL_LEFT) {
                         $side = IL_COL_RIGHT;
                     }
+
+                    if ($this->getColType() === 'pd' && in_array(substr($type, 2), $this->dash_side_panel_settings->getValidModules())) {
```

Comment:
Please use `in_array` only with  `strict = true` as such:
```suggestion
                    if ($this->getColType() === 'pd' && in_array(substr($type, 2), $this->dash_side_panel_settings->getValidModules(), true)) {
```